src/wizards.ts

```diff
@@ -0,0 +1,250 @@
+import { ListItem } from '@material/mwc-list/mwc-list-item';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/openscd/open-scd/pullRequest?prid=7195001)
```suggestion
import { ListItem } from "@material/mwc-list/mwc-list-item";
```